experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/README.md

```diff
@@ -0,0 +1,20 @@
+# Test Case to validate the cspc pool creation is failed when the bloxkdevice is already claimed.
+
+## Description
+   - This test is capable of validate the pool creation is failed when the blockdevice is claimed. 
```

Comment:
This test is capable of validating if the pool creation is failed when the specified blockdevices are already claimed